src/backtick/engine.clj

```diff
@@ -123,14 +123,14 @@
             (db/cron-delete! payload)
             (pop-cron-aux))
           (let [next (to-sql-time (time/plus (to-date-time (:next payload))
-                                             (time/millis (:interval payload))))]
-            (db/queue-insert! {:name (:name payload)
-                               :state "running"
-                               :priority (to-sql-time (time/now))
-                               :data (prn-str [])})
+                                             (time/millis (:interval payload))))
+                inserted (db/queue-insert<! {:name (:name payload)
+                                             :state "running"
+                                             :priority (to-sql-time (time/now))
+                                             :data (prn-str [])})]
```

Comment:
Actually, since you're just passing back this row directly, and not going through `db/queue-pop`, I think you should be manually setting the following fields here as well:
- `started_at`
- `updated_at`
- `tries`